examples/bank/bank.go

```diff
@@ -33,16 +33,26 @@ import (
 	"github.com/cockroachdb/cockroach/util/log"
 )
 
+var dbName = flag.String("db-name", "", "Name/URL of the distributed database backend.")
 var useTransaction = flag.Bool("use-transaction", true, "Turn off to disable transaction.")
 
+// These two flags configure a range of accounts over which the program functions.
+var firstAccount = flag.Int("first-account", 0, "First account in the account range.")
+var numAccounts = flag.Int("num-accounts", 1000, "Number of accounts in the account range.")
+
+var numParallelTransfers = flag.Int("num-parallel-transfers", 100, "Number of parallel transfers.")
+
 // Makes an id string from an id int.
-func makeAccountID(id int) []byte {
-	return []byte(fmt.Sprintf("%09d", id))
+func (bank *Bank) makeAccountID(id int) []byte {
```

Comment:
You should move this method below where the `Bank` struct is defined. Feels odd to see it first.